src/main/resources/jenkins/plugins/http_request/HttpRequest/global.jelly

```diff
@@ -81,20 +82,5 @@
                 </table>
             </f:repeatable>
         </f:entry>
-
-        <f:section title="HTTP Request - Deprecated properties (this properties will be removed soon. They still here to re-apply while saving a job)">
```

Comment:
It is not. I have asked the author if he agrees to actually deprecate the properties on the dev mailing list. I just wanted to have fewer things to worry about, I did not want to carry unnecessary baggage into the workflow.